src/main/java/com/wolox/training/models/Book.java

```diff
@@ -34,7 +34,7 @@
     private String author;
 
     @NotNull
-    private String image;
+    private String image = "-";
```

Comment:
I'm sorry I suggested this, I got confused with the DTO that had the variables initialized with this. Your previous approach for `image` was ok, you can revert this.